report_qweb_signer/README.rst

```diff
@@ -0,0 +1,123 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==================
+PDF reports signer
+==================
+
+This module was written to extend the functionality of report addon to sign
+PDFs using a PKCS#12 certificate.
+
+
+Installation
+============
+
+To install this module, you need to install PortableSigner Java application:
+
+::
```

Comment:
You can put that at the end of previous sentence, it's prettier: *[blabla] Java application::*